client_plugin/drivers/vfile/kvstore/kvstore.go

```diff
@@ -57,13 +57,36 @@ const (
 	VolPrefixGRef                     = "SVOLS_gref_"
 	VolPrefixInfo                     = "SVOLS_info_"
 	VolPrefixClient                   = "SVOLS_client_"
+	VolPrefixStartTrigger             = "SVOLS_start_trigger_"
```

Comment:
I am still not quite clear with VolPrefixStartTrigger and VolPrefixStartMarker. Could you explain when those two values are used?